packages/babel-helper-annotate-as-pure/README.md

```diff
@@ -0,0 +1,5 @@
+# babel-helper-annotate-as-pure
+
+## Usage
+
+TODO
```

Comment:
Cool, we could just document the exported function API.

Reply:
sure thing, just werent sure if everything is ok and i have left documenting this package for the moment the PR gets accepted (code-wise), gonna add docs later today